src/Controls/samples/Controls.Sample.UITests/Controls.Sample.UITests.csproj

```diff
@@ -46,4 +46,13 @@
 
   <Import Project="$(MauiSrcDirectory)Maui.InTree.props" Condition=" '$(UseMaui)' != 'true' " />
 
+  <ItemGroup>
+    <MauiXaml Update="Elements\DragAndDropEvents.xaml">
```

Comment:
We can probably delete these yea?

Reply:
These are being used in the DragAndDropGallery, trying to understand your comment, are you saying they are not worth running?

Reply:
Sorry, I'm just referring to this ItemGroup specifically not the actual XAML files. You should able to remove this ItemGroup and those files will still be included and run.

Reply:
Ah yea good call, not sure why VS added them in this case, will do.